Makefile

```
@@ -160,16 +167,14 @@ nuke: clean devnet-clean ## Completely clean the project directory
 	git clean -Xdf
 .PHONY: nuke
 
-pre-devnet: submodules ## Prepares for running a local devnet
+## Prepares for running a local devnet
+pre-devnet: submodules cannon-prestate
```

Could we also just directly add
```suggestion
pre-devnet: submodules op-program/bin/prestate-proof.json
```
here and then skip the definition of the phony `cannon-prestate` target?

---

I kept cannon-prestate as the phony that wraps all the outputs (yeah there's only one right now but it's not nice to name it from outside the makefile). I think this is better than dropping the phony and leaving documentation and users confused when the phony is very nice and convenient.